examples/appdir/src/app/sink/page.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import { Inter } from "next/font/google";
+import { UploadButton, UploadDropzone } from "@uploadthing/react";
+import type { OurFileRouter } from "../_uploadthing";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export default function Home() {
+  return (
+    <main className="flex min-h-screen flex-col items-center justify-center  gap-16 p-24">
+      <div className="flex flex-col items-center justify-center gap-4">
+        <div className="flex gap-4">
+          <UploadButton<OurFileRouter>
+            endpoint="withoutMdwr"
+            onClientUploadComplete={(res) => {
+              console.log("Files: ", res);
+              alert("Upload Completed");
+            }}
+            onUploadError={(error: Error) => {
+              alert(`ERROR! ${error.message}`);
+            }}
+          />
+
+          <UploadButton<OurFileRouter>
```

Comment:
Is there a reason this isn't just in the main `Page.tsx` example?

Also, pagedir and solid examples?

Reply:
I need a sandbox. I want it somewhere in this repo. I don't need more than one sandbox. tRPC has a "kitchen-sink" example. We can do something like that. Or I can just delete this lol.